spec/factories/request_for_comment.rb

```diff
@@ -13,7 +13,7 @@
     factory :rfc_with_comment, class: 'RequestForComment' do
       after(:create) do |rfc|
         rfc.file = rfc.submission.files.first
-        Comment.create(file: rfc.file, user: rfc.user, text: "comment for rfc #{rfc.question}")
+        Comment.create(file: rfc.file, user: rfc.user, row: 1, text: "comment for rfc #{rfc.question}")
```

Comment:
Could you please quickly explain what the `row` is used for? Does it indicate the row (as in line of code) a comment refers to?

Reply:
Exactly, it refers to the line of code where the comment was made.